<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkAnalyzerGeneralMenu</class>
 <widget class="QWidget" name="NetworkAnalyzerGeneralMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>502</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_23">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="scopy::gui::MenuHeader" name="widgetMenuHeader" native="true"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vLayout_15">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="scopy::gui::SubsectionSeparator" name="widgetPlotSubsSep" native="true"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLayout_17">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lbl_16">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbBoxGraphs">
             <property name="minimumSize">
              <size>
               <width>141</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Bode</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Nyquist</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Nichols</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="vSpacer_13">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>15</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLayout_13">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="vLayout_12">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lbl_10">
               <property name="text">
                <string>Line thickness</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QComboBox" name="cmbBoxLineThickness">
             <property name="minimumSize">
              <size>
               <width>141</width>
               <height>0</height>
              </size>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <item>
              <property name="text">
               <string>0.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="scopy::gui::SubsectionSeparator" name="widgetExportSubsSep" native="true"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnExport">
           <property name="styleSheet">
            <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;

  border: 0px;
  font-size: 14px;
}</string>
           </property>
           <property name="text">
            <string>Export</string>
           </property>
           <property name="blue_button" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="scopy::gui::SubsectionSeparator" name="widgetReferenceSubsSep" native="true"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vLayout_20">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineEditImportFile">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="vSpacer_22">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnImport">
             <property name="styleSheet">
              <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;

  border: 0px;f;

  font-size: 14px;
}</string>
             </property>
             <property name="text">
              <string>Import</string>
             </property>
             <property name="blue_button" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="vSpacer_21">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnSnapshot">
             <property name="styleSheet">
              <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;

  border: 0px;

  font-size: 14px;
}</string>
             </property>
             <property name="text">
              <string>Snapshot</string>
             </property>
             <property name="blue_button" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="vSpacer_20">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnRemoveReference">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;

  border: 0px;
  font-size: 14px;
}</string>
             </property>
             <property name="text">
              <string>Remove Reference</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="blue_button" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="vSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>scopy::gui::SubsectionSeparator</class>
   <extends>QWidget</extends>
   <header location="global">scopy/gui/subsection_separator.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>scopy::gui::MenuHeader</class>
   <extends>QWidget</extends>
   <header location="global">scopy/gui/menu_header.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
